dagshub/common/cli.py

```diff
@@ -145,6 +170,12 @@ def to_log_level(verbosity):
 @click.option("-v", "--verbose", default=0, count=True, help="Verbosity level")
 @click.option("-q", "--quiet", is_flag=True, help="Suppress print output")
 @click.option("--host", help="DagsHub instance to which you want to login")
+@click.option(
+    "--bucket",
+    is_flag=True,
+    help="Download the file(s) to the repo's DagsHub Storage bucket. "
```

Comment:
A link to docs would be nice if possible
```suggestion
    help="Download the file(s) from the repo's DagsHub Storage bucket. "
```